### ISSUE TITLE
Camera not properly recognised on Raspbian Buster 

### ISSUE THREAD

**Issue by Pi-DD** [ccrisan/motioneye#1351]
See this item for background information....
https://github.com/ccrisan/motioneye/issues/1348#issue-462824686

I recognise this is “bleeding edge” since Buster very new and its base of Debian 10 not out for a few days!!,

Installation on Buster seems possible, as described in the link, but when installed

1. Version information is missing from General Settings
2. Camera is not fully recognised (locally attached Pi Noir) when it is recognised fully on Stretch. Running the camera direct from Raspivid works OK. 

See attached
![11556750-2B29-4A73-BE9C-A72CD8BD8C4F](https://user-images.githubusercontent.com/38586714/60512954-e2465680-9ccd-11e9-9a39-4e892378ade7.png)
![DAE5A14F-8C88-4C15-9A77-1926E8F1750E](https://user-images.githubusercontent.com/38586714/60512955-e2465680-9ccd-11e9-9fa5-dedf725a078a.png)

First screenshot from Buster, second from Stretch. 
Note missing version info in general settings

In both cases camera configured as mmal camera followed by the only option offered of Videocore Camera. 

Buster version white balance never adjusts and no video resolution or other options appear. 

Tried turning off camera config, reboot, turn on, reboot - makes no difference.

PS - I've tried the simple expedient of re-booting the Pi Zero from an SD with Stretch + Motioneye to make sure it is not a cable/hardware problem with the Zero. It's not. Everything works perfectly on the Zero when booted that way.

**Comment by ccrisan** [comment]
Enable advanced settings on Buster and you should be fine.

**Comment by Pi-DD** [comment]
Calvin, thanks for that! User error not spotting it! 

Turning it on reveals detail but purple colour cast remains. Turning on Auto Brightness puts something into a loop of brightening and dimming that never settles to a specific state, all with a purple overcast. 

When I have time I’ll investigate further within the limits of my skill!

**Comment by Pi-DD** [comment]
Turns out this is a known problem in the Pi AWB algorithm that was changed recently and does not handle the NOIR case. There is a workaround.

https://github.com/raspberrypi/firmware/issues/1167
https://www.raspberrypi.org/forums/view ... 3&t=241633

Sorry to have pointed this at the excellent MotionEye when (a) I made a user error in not noting Advanced Settings and (b) my adventure of Buster install gave me the mindset it was a MotionEye issue!
